tests/album-delete.test.js

```diff
@@ -0,0 +1,42 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('Delete Album', () => {
+  let album;
+  beforeEach(async () => {
+    const { rows } = await db.query(
+      'INSERT INTO Albums (name, year) VALUES ($1, $2) RETURNING *',
+      ['Portrait With Firewood', 2018]
+    );
+
+    album = rows[0];
+  });
+
+  describe('DELETE /albums/${id}', () => {
+    it('deletes the album and returns the deleted data', async () => {
+      const { status, body } = await request(app)
+        .delete(`/albums/${album.id}`)
+        .send();
+
+      expect(status).to.equal(200);
+
+      expect(body).to.deep.equal({
+        id: album.id,
+        name: 'Portrait With Firewood',
+        year: 2018,
+        artistid: null,
```

Comment:
If you implemented the above suggestion you could then check that the artistid has been entered correctly, probably worthwhile as it comes from the req.params instead of the req.body - it would be testing that part of the code too